templates/parachain/pallets/template/Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
 name = "pallet-parachain-template"
-authors = ["Anonymous"]
+version = "4.0.0-dev"
```

Comment:
If we dont publish it then it could also be `0.1.0`.

Reply:
Everything in templates should be no publish and I will put all versions to `0.1.0` or `0.0.0`